src/d3d12/nvapi_d3d12_device.h

```diff
@@ -6,6 +6,7 @@
 
 namespace dxvk {
 
+    typedef ID3D12GraphicsCommandListExt1 d3d12_graphics_command_list_ext_iface;
```

Comment:
I had done this to reduce the number of lines we'd need to change if another interface version was written in the future. I'll remove it from my changes since it's not really necessary.